pkg/reconciler/labeler/v1/accessors.go

```diff
@@ -0,0 +1,123 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1
+
+import (
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/types"
+	"knative.dev/pkg/kmeta"
+
+	"knative.dev/serving/pkg/apis/serving"
+	clientset "knative.dev/serving/pkg/client/clientset/versioned"
+	listers "knative.dev/serving/pkg/client/listers/serving/v1"
+)
+
+// Accessor defines an abstraction for manipulating labeled entity
```

Comment:
It's strange that a public interface has no public methods.
But I guess once the migration is over we can reconcile all these interfaces.

Reply:
that's why initially I had made it private even though a function returns it (so that it may only be used to make a new one and pass to labeler.go). This really all exists to get the lister off the reconciler